internal/storage/metricstore/elasticsearch/reader.go

```diff
@@ -108,15 +126,17 @@ func (r MetricsReader) GetCallRates(ctx context.Context, params *metricstore.Cal
 		boolQuery:           r.buildQuery(params.BaseQueryParameters, timeRange),
 		aggQuery:            r.buildCallRateAggregations(params.BaseQueryParameters, timeRange),
 		bucketsToPointsFunc: getCallRateBucketsToPoints,
-		processMetricsFunc:  getCallRateProcessMetrics,
 	}
 
-	metricFamily, err := r.executeSearch(ctx, metricsParams)
+	rawMetricFamily, err := r.executeSearch(ctx, metricsParams)
 	if err != nil {
 		return nil, err
 	}
+
+	// Process the raw aggregation value to calculate call_rate (req/s)
+	processedMetricFamily := getCallRateProcessMetrics(rawMetricFamily, params.BaseQueryParameters)
```

Comment:
wouldn't calcCallRate be more accurate and relevant name?
```suggestion
	processedMetricFamily := calcCallRate(rawMetricFamily, params.BaseQueryParameters)
```

Reply:
Yes, it makes sense. Thanks for your feedback